frontend/src/Components/Map.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import { tileNames, tilesMap } from "./tilesMap.js";
+import "./Map.css";
+
+class Map extends React.Component {
+  shouldComponentUpdate() {
```

Comment:
The behavior will prevent your map from updating. It will prove a bad idea when you try and create a second map

Reply:
Thank you for your feedback, for the next map we are going to load in the game, we are going to implement a boolean for geting true this method.